scarb/tests/proc_macro_v2_expand.rs

```diff
@@ -1034,7 +1034,7 @@ fn code_mappings_preserve_attribute_error_on_inner_trait_locations_with_parser()
                     ^^^^^
             note: this error originates in the attribute macro: `some`
             
-            error: could not compile `hello` due to previous error
+            error: could not compile `hello` due to 1 previous error
```

Comment:
```suggestion
            error: could not compile `hello` due to [..] previous error
```

would reduce test maintenance burden perhaps?